studybuddies/src/app/services/token-storage.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
import { Injectable } from "@angular/core";
```